model/src/main/java/com/dremio/nessie/model/Transplant.java

```diff
@@ -35,5 +35,4 @@
 
   List<String> getHashesToTransplant();
```

Comment:
Looks like you decide to punt on range for this patch?

Reply:
I wanted to wrap python tests first